engine/api/pipeline_job.go

```diff
@@ -155,6 +159,10 @@ func (api *API) updateJobHandler() Handler {
 		}
 		defer tx.Rollback()
 
+		if err := pipeline.CreateAudit(tx, pipelineData, "updateJob",getUser(ctx)); err != nil {
```

Comment:
add this kind of constant ("updateJob") in const in sdk in order to not save a mispell const in database